app.js

```diff
@@ -202,7 +209,7 @@ function run(keys) {
 
                 if (data[0].endsWith('OnError')) {
                     // monkey-patch java/swift sdk bugs.
-                    data[0] = data[0].replace('OnError', 'OnFailure');
```

Comment:
@ggarber can you check if the sdk has been patched in theory? Then we can just remove this hack instead